src/main/java/com/thirdcc/webapp/service/impl/EventServiceImpl.java

```diff
@@ -111,4 +116,24 @@ public Event findEventByIdAndNotCancelledStatus(Long id) {
         return eventRepository.findOneByIdAndStatusIn(id, eventStatuses)
             .orElseThrow(() -> new BadRequestException("Event not found, might be cancelled or not exist"));
     }
+
+    /**
+     * Cancel event by id.
+     *
+     * @param id the id of the entity.
+     */
+    public EventDTO cancelEventById(Long id) {
```

Comment:
Missing Override Annotation

Reply:
Resolved in fc1c175c9b7fbfb27dbf8eb8463c2f09acd3088b